Application/Features/SendRecurringInfoFeature/Queries/GetCurrencySymbolsListQuery.cs

```diff
@@ -11,32 +11,69 @@ public class GetCurrencySymbolsListQueryRequest : IRequest<GetCurrencySymbolsLis
 
     public class GetCurrencySymbolsListQueryHandler : IRequestHandler<GetCurrencySymbolsListQueryRequest, GetCurrencySymbolsListQueryResponse>
     {
+        private readonly HttpClient _httpClient;
+        private readonly string _apikey = "uHnC6f1zjtWiQbIyjUAcdu48L2uoom8V";
+
+        public GetCurrencySymbolsListQueryHandler(IHttpClientFactory httpClientFactory)
+        {
+            _httpClient = httpClientFactory.CreateClient();
+            _httpClient.DefaultRequestHeaders.Add("apikey", _apikey);
+        }
+
         public async Task<GetCurrencySymbolsListQueryResponse> Handle(GetCurrencySymbolsListQueryRequest request, CancellationToken cancellationToken)
         {
             var symbols = new Currencies();
             var properties = symbols.GetType().GetProperties();
             var currencySymbols = new List<string>();
-                
+
             for (int i = 0; i < 10; i++)
             {
                 currencySymbols.Add(properties[i].Name);
             }
+            
+            List<ExchangeRateInfo> ExchangeRates = new();
+
             if (request.CurrencySymbol is not null)
             {
                 currencySymbols.Remove(request.CurrencySymbol.ToUpper());
-            }
+                var httpResponse = await _httpClient.GetAsync($"https://api.apilayer.com/currency_data/live?source={request.CurrencySymbol}&currencies={"USD,EUR,RUB,AED,KWN,KZT,LAK"}");
+                httpResponse.EnsureSuccessStatusCode();
```

Comment:
the same here url is static so write them in the config and pull them from config